src/components/Plot.js

```diff
@@ -1,115 +1,221 @@
+                  {plot.persons.length<1 ? <p>No linked characters</p> : <ul> {relatedCharacters.map(p => <li key={p.id}>
+                    <Link onClick={() => props.changeTab('Characters')} to={`/characters/${p.id}`}>{p.full_name}</Link>
+                    <span> - {p.is_character ? "Character" : "NPC"}</span></li>)}
+                    </ul>
+                  }</Col>
+              </Row>
+              <Row>
+                <Col sm><span className='mini-header'>Character Groups Involved</span></Col>
+              </Row>
+              {characterGroups.length <1 ? <p>No linked character groups</p> : 
+              <ul>{characterGroups.map(g => <li key={g}><Row><Col sm>{g}</Col></Row></li>)}</ul>}
+              <Row>
+                <Col sm><span className='mini-header'>Basic Info</span></Col>
+              </Row>
+              <Row>
+                <Col sm={6}><span className='caption'>GM Actions: </span>{plot.gm_actions}</Col>
+                <Col sm={4}><span className='caption'>Plot size: </span>{plot.size}</Col>
+              </Row>
+              <Row>
+                <Col sm={6}><span className='caption'>Text NPC should send first message: </span>{plot.text_npc_first_message ? "Yes" : "No"}</Col>
+                <Col sm={4}><span className='caption'>Plot themes: </span>
+                {plot.themes.length<1 ? <span>No themes defined</span> : plot.themes}</Col>
+              </Row>
+              <Row>
+                <Col sm={6}><span className='caption'>Happens after jump: </span>
+                {plot.after_jump ? plot.after_jump : <span>No jump defined</span>}
+                <span className='new'> (Editable unless plot is locked)</span></Col>
+                <Col sm={4}><span className='caption'>Plot Importance: </span> {plot.importance}</Col>
+              </Row>
+              <Row>
+                <Col sm={6}><span className='caption'>Locked plot: </span>{plot.locked ? "Yes" : "No"}</Col>
+              </Row>
+              <Row>
+                <Col sm>&nbsp;</Col>
+              </Row>
+              <Row>
+                <Col sm><span className='mini-header'>Events <span className='new'>[CREATE EVENT BUTTON]</span></span></Col>
+              </Row>
+              {plot.events.length<1 ? <p>No linked events</p> : <ul> {plot.events.map(e => <li key={e.id}>
+                  <Link onClick={() => props.changeTab('Events')} to={`/events/${e.id}`}>{e.name}</Link></li>)}
+                  </ul>
+              }
+              <Row>
+                <Col sm><span className='mini-header'>Messages <span className='new'>[CREATE MESSAGE BUTTON]</span></span></Col>
+              </Row>
+              {relatedMessages.length<1 ? <p>No messages</p> : <ul> {relatedMessages.map(m => <li key={m.id}>
+                  <Link onClick={() => props.changeTab('Messages')} to={`/messages/${m.id}`}>{m.name}</Link> {m.sent ? "[Already sent]" : "[Not send yet"}</li>)}
```

Comment:
m.sent is not boolean, it is a string. Now it always returns true. This can be only [{m.sent}].

Reply:
An excellent catch! Thank you :) Fixed it. Went with "Sent: {m.sent}" for clarity.